registries/hello_world.hocon

```diff
@@ -41,14 +50,14 @@ Tell me what your target audience is, and what sentiment you would like to relat
 
             "instructions": """
 You are an author of terse announcements.
-You will be asked to help writing an extremely terse announcement on behalf of a person or organization.
+You will be asked to help writing the shortest possible announcement on behalf of a person or organization.
```

Comment:
Use the latest hello_world.hocon from neuro-san.
Some improvements have been made to the prompts over time.